format/format.go

```diff
@@ -83,6 +83,7 @@ var (
 	Bitcoin_Block       = &decode.Group{Name: "bitcoin_block"}
 	Bitcoin_Script      = &decode.Group{Name: "bitcoin_script"}
 	Bitcoin_Transaction = &decode.Group{Name: "bitcoin_transaction"}
+	OpenTimestamps      = &decode.Group{Name: "ots"}
```

Comment:
I prefer to keep the names in sync so could the name `open_timestamps`?

Reply:
Can it be `opentimestamps`?